### DISCUSSION TITLE
VPN on TCP 443

### DISCUSSION THREAD

**Discussion by ptchuba** [strongswan/strongswan#2274] [General]
I have some users that try to connect from environments where IPSEC is not allowed outbound. As usual it will be great to use some tunneling on 443. I understand that I can change the default strongswan ports but if I'm not mistaken, any good firewall should be able to detect that this is still IPSEC and block it accordingly. I wonder what you guys are doing in these types of situations. Are you simply adding some other protocol like openvpn or do you have some custom way to make this work with Strongswan? It would be great to have something like what Cisco does with their anyconnect which supports both dtls and ipsec.

Will be glad to hear your thoughts.

**Comment by gongoscho** [comment]
Hello,
i have solved it like this.

Since I cannot change the IPSec NAT-T Port: 4500 UDP via the GUI in the firewall: OPNsense, a NAT is created on WAN: HTTP 80 UDP to NAT-T 4500 UDP.

Enable NAT reflection

Same with Port 443 UDP(QUIC) possible.

grEEtZ
